lib/inputs/file-pattern/static.ts

```diff
@@ -34,7 +34,7 @@ export function generateExtractAction(
   return {
     actionName: "find-files-by-pattern",
     filePathMatches: generatePathMatcher(globsInclude, globsExclude),
-    callback: streamToString,
+    callback: streamToBuffer,
```

Comment:
The content of manifest files is not streamed into a buffer.

Reply:
how come we don't want the manifest files as base64 encoded str any more?

Reply:
This is based on this comment from @Shesekino in slack: https://snyk.slack.com/archives/CBXS33GJY/p1589820258095800?thread_ts=1589385083.016900

... if you care about the context and want to go down the :rabbit: :hole: have a look at the whole thread - but be warned it is huge :sweat_smile: